liquidator/detail/score_matrix_detail.h

```diff
@@ -79,10 +79,10 @@ scale(const PWM& pwm, const std::pair<double, double>& min_max, const unsigned r
 }
 
 // returns score; sequences with invalid characters return 0.
-unsigned score(const std::vector<std::array<unsigned, AlphabetSize>>& matrix,
-               const std::string& sequence,
-               const size_t begin,
-               const size_t end)
+inline unsigned score(const std::vector<std::array<unsigned, AlphabetSize>>& matrix,
```

Comment:
First page appears to back me up, external linkage is what we're dealing with. Second page is mixing up static functions and static variables.